api/task_processor/processor.py

```diff
@@ -89,6 +90,13 @@ def _run_task(task: Task) -> typing.Optional[typing.Tuple[Task, TaskRun]]:
 
         task_run.finished_at = timezone.now()
         task.mark_success()
+    except DatabaseError as e:
+        logger.error(
+            "Database error while running task %s:  %s",
+            task.id,
+            e,
+            exc_info=True,
+        )
```

Comment:
I think we have to add `task.mark_falure()` here?

Reply:
No, there is no point doing that since we don't have access to the database any more here